### ISSUE TITLE
TypeError: Cannot read property 'filter' of undefined

### ISSUE THREAD

**Issue by sjparkinson** [Financial-Times/nori#106]
When creating a new session called `bower-1.8.8`.

```
(node:1537) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'filter' of undefined
    at MultiSelect.filter (/Users/samuel.parkinson/Repositories/Financial-Times/nori/node_modules/enquirer/lib/types/array.js:509:26)
    at MultiSelect.get enabled [as enabled] (/Users/samuel.parkinson/Repositories/Financial-Times/nori/node_modules/enquirer/lib/types/array.js:620:17)
    at MultiSelect.get selected [as selected] (/Users/samuel.parkinson/Repositories/Financial-Times/nori/node_modules/enquirer/lib/types/array.js:636:33)
    at MultiSelect.submit (/Users/samuel.parkinson/Repositories/Financial-Times/nori/node_modules/enquirer/lib/types/array.js:546:22)
    at Promise (/Users/samuel.parkinson/Repositories/Financial-Times/nori/node_modules/enquirer/lib/prompt.js:234:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:1537) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:1537) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by quarterto** [comment]
can't reproduce this on master with a fresh `node_modules`

**Comment by apaleslimghost** [comment]
this is reproducible when running `npx nori`. seems to be coming from deep inside `inquirer`, updating that might fix it?

@andygout this is currently blocking people using nori, it'd be great to have this fixed before cutting 2.0.0 final 🙏

**Comment by apaleslimghost** [comment]
doesn't seem to be happening on 2.0.0-beta.11, can anyone else repro it?
